### DISCUSSION TITLE
Στα κείμενα με τις ερωτήσεις δεν εμφανίζεται το κείμενο

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/hci#1615] [Announcements]
Εκεί που κάνουμε τα κείμενα με τις ερωτήσεις δεν εμφανίζεται το κείμενο που έχουμε να διαβάσουμε αυτή την εβδομάδα. Τι γίνεται;

**Comment by epidrome** [comment]
Ας μας πουν και οι άλλοι αν έχουν τέτοιο πρόβλημα, γιατί εγώ το βλέπω κανονικά, μήπως είναι τεχνικό πρόβλημα στο δικό σας τερματικό;

**Reply by Sola1r3** [reply]
Και εμένα μου τα εμφανίζει κανονικά.
